default.yml

```diff
@@ -12,7 +12,10 @@ AllCops:
 Bundler/OrderedGems:
   Enabled: false
 
-Layout/AlignParameters:
+Layout/ArgumentAlignment:
```

Comment:
This is a new cop. Enabled using `with_fixed_indentation` for consistency with `ParameterAlignment` (and also with the api codebase).

`AlignParameters` was renamed to `ParameterAlignment` below

Reply:
I agree that `with_fixed_indentation` is the correct choice (I'm surprised that this isn't the default).